recipe/bld.bat

```diff
@@ -24,3 +24,5 @@ copy prog\udunits2.exe %SCRIPTS%\udunits2.exe || exit 1
 copy lib\udunits2.dll %SCRIPTS%\udunits2.dll || exit 1
 copy lib\*.exp %LIBRARY_LIB%\ || exit 1
 copy lib\*.lib %LIBRARY_LIB%\ || exit 1
+
+dir %LIBRARY_PREFIX%\share\udunits\
```

Comment:
@msarahan in line `9` we copy the `xml` files to the `%LIBRARY_PREFIX%\share\udunits\` and they are present as this test shows: https://ci.appveyor.com/project/conda-forge/udunits2-feedstock/build/1.0.41/job/j57npt3gu069srvm#L516

However, `AppVeyor` fails b/c `conda-build` cannot find them later: https://ci.appveyor.com/project/conda-forge/udunits2-feedstock/build/1.0.41/job/j57npt3gu069srvm#L590

What am I doing wrong? If I downgrading to `conda-build` to `2.1.1` works, so I suspect it is a change that I missed in the latest version.